_posts/0000-01-02-Darkcoderhacker.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
A true friend is someone who is there for you when he’d rather be anywhere else.